examples/default/index.js

```
@@ -11,73 +11,85 @@
 */
 const { api, data, schedule } = require("@serverless/cloud"); // eslint-disable-line
 
-
-/* 
+/*
  * Create a route to GET our TODO items
-*/
-api.get('/todos', async (req, res) => {
-
-  // Call our getTodos function with the status
-  let result = await getTodos(req.query.status, req.query.meta ? true : {});
-
-  // Return the results
-  res.send({
-    items: result.items
-  })
-})
+ */
+api.get("/todos", async (req, res, next) => {
+  try {
```

These aren't necessary, as the runtime does it for you (currently via express-async-errors)

---

I've found them necessary when building apps for the cloud hackathon. Errors were not handled without them 🤔

---

I think that should be classified as a bug then!

---

hmm ok, feel free to close this PR then 👌

---

I was thinking the same thing. They should be handled for you, and if you have error middleware, you can control the output.